### ISSUE TITLE
WARNING:whipper.command.main:set_hostname() takes 1 positional argument but 2 were given

### ISSUE THREAD

**Issue by texneus** [whipper-team/whipper#464]
I am getting subject warning immediately each time whipper is run.  Command parameters do not seem to matter.  Seems harmless (everything still appears to work) but don't see it reported so I'm not sure if I'm the only one?  I'm running 0.9.0 pulled and built from Git as per the README.MD in MX Linux 19.

Ex:
```
$ whipper
WARNING:whipper.command.main:set_hostname() takes 1 positional argument but 2 were given
usage: whipper [-R] [-v] [-h] [-e {never,failure,success,always}]

whipper is a CD ripping utility focusing on accuracy over speed.

whipper gives you a tree of subcommands to work with.
You can get help on subcommands by using the -h option to the subcommand.

optional arguments:
  -R, --record          record API requests for playback
  -v, --version         show version information
  -h, --help            show this help message and exit
  -e {never,failure,success,always}, --eject {never,failure,success,always}
                        when to eject disc (default: success)

commands:
  accurip  handle AccurateRip information
  cd       handle CDs
  drive    handle drives
  image    handle images
  mblookup lookup MusicBrainz entry
  offset   handle drive offsets
```

**Comment by ABCbum** [comment]
Hi @texneus , this is happening due to your local version of musicbrainzngs doesn't support using https (check out this code https://github.com/whipper-team/whipper/blob/9e63915f65017408d6f12b22f7ba96bbb866cf5e/whipper/command/main.py#L28). 
I think maybe whipper should give a better warning message.

**Comment by JoeLametta** [comment]
Thanks for reporting this issue. Yeah, this is harmless unless you define a custom MusicBrainz server which is HTTPS only.
The issue should now be fixed (clearer warning) with commit 9a0b911666f5f33cc2cdda90b6041f3fe259745a.
